.github/workflows/changelog.yml

```diff
@@ -0,0 +1,37 @@
+---
+## Workflow to check CHANGELOG was updated when version gets updated in share/version.py
+name: changelog
+
+on:
+  pull_request:
+    paths:
+      - 'share/version.py'
+
+jobs:
+
+  changelog:
+    runs-on: ubuntu-latest
+    timeout-minutes: 5
+
+    steps:
+
+      - uses: actions/checkout@v4
+
+      - name: Get version number
+        shell: bash
+        run: |
+          VERSION=$(awk '/version/ {print $3}' share/version.py | tr -d '"')
```

Comment:
Consider 
shell grep -oE '[0-9]+\.[0-9]+\.[0-9]+(\-[a-zA-Z]+[0-9]+)?' /share/version.py

https://github.com/elastic/elastic-agent/blob/main/deploy/kubernetes/Makefile#L5C16-L5C74

(I am afraid of someone putting an extra space in file and breaks the automation)

Reply:
Can you explain this part 

> I am afraid of someone putting an extra space in file and breaks the automation

a bit more @gizas? Maybe by an example?  I don't understand why it would break

Reply:
version = " 1.13.1" 

then

```bash
awk '/version/ {print $4}' version.py | tr -d '"'  ---> Thins needs to become $4
1.13.1
❯ grep -oE '[0-9]+\.[0-9]+\.[0-9]+(\-[a-zA-Z]+[0-9]+)?' version.py ---> this still works
1.13.1
```

Reply:
Thank you! I checked and you are corrected. I updated it.